pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/defaultvalues/BasicDefaultDataMaker.java

```diff
@@ -28,7 +28,8 @@ public abstract class BasicDefaultDataMaker implements Defaultable {
     protected ElasticSearchParticipantDto elasticSearchParticipantDto;
 
 
-    protected abstract boolean setDefaultData();
+    @VisibleForTesting
+    abstract boolean setDefaultData();
```

Comment:
Is this actually used in a test somewhere?

Reply:
No--I forgot to undo this as I experimented with different approaches.